app/move/controllers/create/prison-transfer-reason.test.js

```diff
@@ -0,0 +1,71 @@
+const fieldHelpers = require('../../../../common/helpers/field')
+const referenceDataService = require('../../../../common/services/reference-data')
+
+const FormController = require('hmpo-form-wizard').Controller
+
+const PrisonTransferReasonsController = require('./prison-transfer-reason')
+const prisonTransferReasonsController = new PrisonTransferReasonsController({
+  route: '/prison_transfer_reason',
+})
+
+describe('Prison transfer reason controller', function() {
+  let stubForMapReferenceDataToOption
+  let stubForGetPrisonTransferReasons
+  let stubForParentConfigure
+  let mockRequest
+
+  context('happy path', function() {
+    beforeEach(async function() {
+      mockRequest = {
+        form: {
+          options: {
+            fields: {
+              prison_transfer_reason: {},
+            },
+          },
+        },
+      }
+      stubForParentConfigure = sinon.stub(FormController.prototype, 'configure')
+      stubForMapReferenceDataToOption = sinon
+        .stub(fieldHelpers, 'mapReferenceDataToOption')
+        .returns({})
+      stubForGetPrisonTransferReasons = sinon
+        .stub(referenceDataService, 'getPrisonTransferReasons')
+        .resolves([{}, {}])
+      await prisonTransferReasonsController.configure(mockRequest, {}, () => {})
+    })
+    it('calls reference data service', function() {
+      expect(stubForGetPrisonTransferReasons).to.have.been.calledOnce
+    })
+    it('appends conditionals to the correct fields', function() {
+      expect(stubForMapReferenceDataToOption).to.have.been.calledTwice
+    })
+    it('sets the correct reason items', function() {
+      expect(
+        mockRequest.form.options.fields.prison_transfer_reason.items
+      ).to.deep.equal([{}, {}])
+    })
+    it('invokes the parent configure', function() {
+      expect(stubForParentConfigure).to.have.been.calledOnce
```

Comment:
This can also just be:

```javascript
expect(FormController.prototype,.configure).to.have.been.calledOnce
```

Then no need to define an extra variable.